docker-compose.yml

```diff
@@ -128,15 +128,57 @@ services:
     networks:
       - djangonetwork
 
-
-  mailhog:
-    image: mailhog/mailhog:v1.0.0
-    container_name: scoap3_local_mailhog
+  matomo:
+    image: matomo:4.15
+    restart: 'always'
+    container_name: scoap3_local_matomo
+    environment:
+      - MATOMO_DATABASE_HOST=db
+    depends_on:
+      - matomo-db
+    volumes:
+      - matomo:/var/www/html:z
+    env_file:
+      - .envs/.django
     ports:
-      - '8025:8025'
+      - 8080:80
     networks:
       - djangonetwork
 
+  matomo-db:
+    image: mariadb:10.11
+    command: --max-allowed-packet=64MB
+    container_name: scoap3_local_matomo_db
+    restart: 'always'
+    volumes:
+      - matomo-db:/var/lib/mysql:Z
+    environment:
+      - MARIADB_AUTO_UPGRADE=1
+      - MARIADB_DISABLE_UPGRADE_BACKUP=1
+    env_file:
+      - .envs/.django
+    networks:
+      - djangonetwork
+
+  prometheus:
+    image: prom/prometheus:v2.14.0
+    volumes:
+      - prometheus:/etc/prometheus/:Z
+    ports:
+      - 9090:9090
+
+  grafana:
+    image: grafana/grafana-enterprise
+    container_name: grafana
+    restart: unless-stopped
+    ports:
+     - '4000:3000'
```

Comment:
I would do the same, remove it from local env. Since the settings are different, we can not test the actual dashboard from production.